SonarQube.Client/Requests/PagedRequestBase.cs

```diff
@@ -33,8 +33,8 @@ namespace SonarQube.Client.Requests
     /// <typeparam name="TResponseItem">The type of the items returned by this request.</typeparam>
     public abstract class PagedRequestBase<TResponseItem> : RequestBase<TResponseItem[]>, IPagedRequest<TResponseItem>
     {
-        private const int FirstPage = 1;
-        private const int MaximumPageSize = 500;
+        protected const int FirstPage = 1;
+        protected const int MaximumPageSize = 500;
```

Comment:
Do these need to be `protected`? They don't appear to be used outside this class.

Reply:
This is no longer needed! I wasn't pleased with my first solution and forgot to revert this bit.